sdks/python/apache_beam/transforms/environments.py

```diff
@@ -203,7 +223,7 @@ def from_runner_api_parameter(payload, context):
   def from_options(cls, options):
     config = json.loads(options.environment_config)
     return cls(config.get('command'), os=config.get('os', ''),
-               arch=config.get('arch', ''), env=config.get('env', ''))
+               arch=config.get('arch', ''), env=config.get('env'))
```

Comment:
yeah, I have a fix for this in my typing branch.   fine to fix it here as long as the reviewers agree.